include/core/variableAttributes.h

```diff
@@ -86,4 +86,6 @@ struct variableAttributes
   eval_flags_for_eq_type(const variableAttributes &other_variable);
 };
 
+using AttributesList = std::map<uint, variableAttributes>;
```

Comment:
Move to beginning, line 12

Reply:
that doesn't make sense because that class isn't defined yet